compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -153,6 +153,24 @@ class MultiOutputDetector final : public luci::CircleNodeMutableVisitor<bool>
   bool visit(luci::CircleUnpackOut *) final { return true; }
   bool visit(luci::CircleWhileOut *) final { return true; }
 
+  bool visit(luci::CircleBidirectionalSequenceLSTMOut *) final
+  {
+    // store_outputs(node, 1);
+    return true;
+  }
+
+  bool visit(luci::CircleBidirectionalSequenceLSTM *node) final
+  {
+    if (node->merge_outputs())
+    {
+      store_outputs(node, 1);
+    }
+    else
+    {
+      store_outputs(node, 2);
+    }
+    return true;
+  }
```

Comment:
please add an empty space after this line

Reply:
will address this, thanks.